libs/sdk-core/src/breez_services.rs

```diff
@@ -237,7 +237,7 @@ impl BreezServices {
         req: SendPaymentRequest,
     ) -> Result<SendPaymentResponse, SendPaymentError> {
         self.start_node().await?;
-        let parsed_invoice = parse_invoice(req.bolt11.as_str())?;
+        let parsed_invoice = parse_invoice(req.invoice.as_str())?;
```

Comment:
I believe this still parses a bolt11 invoice?

Reply:
`pay_offer` internally calls `send_payment`, so I thought it would be appropriate to change the field to `invoice` as we now know it can support both bolt11 and 12.

Reply:
I mean the `parse_invoice` function expects a bolt11 invoice. So if a bolt12 invoice is passed here, the parsing will fail.

Reply:
Right. I think the best solution would be to make `parse_invoice` compatible with either, returning either a bolt11 or a bolt12 invoice. Alternatively, I can just replicate the `send_payment` flow on `pay_offer` and create a separate parsing method for bolt 12 invoices. Let me know what you think